core-android/src/test/java/com/uber/sdk/android/core/auth/AuthUtilsTest.java

```diff
@@ -237,25 +234,36 @@ public void generateAccessTokenFromUrl_whenValidAccessTokenWithMultipleScopes_sh
         assertThat(accessToken.getTokenType()).isEqualTo(BEARER);
     }
 
+    @Test
+    public void isAuthorizationCodePresent_whenPresent_shouldReturnTrue() {
+        String redirectUrl = "http://localhost:1234?code=" + AUTH_CODE;
+
+        assertTrue(AuthUtils.isAuthorizationCodePresent(Uri.parse(redirectUrl)));
+    }
+
+    @Test
+    public void isAuthorizationCodePresent_whenEmpty_shouldReturnFalse() {
+        assertFalse(AuthUtils.isAuthorizationCodePresent(Uri.parse("http://localhost:1234?code=")));
+    }
+
+    @Test
+    public void isAuthorizationCodePresent_whenMissing_shouldReturnFalse() {
+        assertFalse(AuthUtils.isAuthorizationCodePresent(Uri.parse("http://localhost:1234")));
+    }
+
     @Test
     public void getCodeFromUrl_whenValidAuthorizationCodePassed() throws LoginAuthenticationException {
         String redirectUrl = "http://localhost:1234?code=" + AUTH_CODE;
 
         assertThat(AuthUtils.parseAuthorizationCode(Uri.parse(redirectUrl))).isEqualTo(AUTH_CODE);
     }
 
-    @Test
+    @Test(expected = LoginAuthenticationException.class)
```

Comment:
nit: I like to assert something about this, such as a partial message

Reply:
Would you prefer a try/catch in the test as the prior implementation used?

Reply:
yeah. Good/bad example https://github.com/uber/AutoDispose/blob/6e1399512f132bc5052c4b1ba111debb6eb430e1/android/autodispose-android/src/test/java/com/uber/autodispose/android/AutoDisposeAndroidPluginsTest.java#L56